Source/VCGeneration/VCGen.cs

```diff
@@ -351,7 +352,7 @@ bool CheckUnreachable(Block cur, List<Cmd> seq)
             }
 
             ch.BeginCheck(cce.NonNull(impl.Name + "_smoke" + id++), vc, new ErrorHandler(absyIds, this.callback),
-              CommandLineOptions.Clo.SmokeTimeout, CommandLineOptions.Clo.ResourceLimit);
+              CommandLineOptions.Clo.SmokeTimeout, CommandLineOptions.Clo.ResourceLimit, cancellationToken);
```

Comment:
Why not `this.cancellationToken`?

Reply:
Actually, the cancellationToken was no longer needed here. I removed it.